### ISSUE TITLE
(bug) 

### ISSUE THREAD

**Issue by vk2r** [relagit/relagit#48]
### What happened?

RelaGit does not show the menus to configure the application in Windows 11

I have recently installed the application in Windows 11 in its version 0.16.8 and unfortunately I do not see the menu to configure the application.

As an additional measure, it would be good to have some key combination for the configuration menu to appear.

![Current](https://github.com/user-attachments/assets/3f47a39e-815d-4ff8-8524-75bb5db3afdd)

![Expected](https://github.com/user-attachments/assets/3aa4add5-e8ef-4a3f-8579-e56925bfc75c)

### Version

0.16.8

### What systems are you seeing the problem on?

Windows

### Confirmations

- [X] I have searched open and closed issues for duplicates
- [X] I have read the relevant documentation
- [X] I have read the CONTRIBUTING document
- [X] I have read the CODE_OF_CONDUCT document

**Comment by BoscoDomingo** [comment]
Yep, can confirm. If you don't know that `Ctrl + ,` brings out the settings, you'd not be able to find them
